frontend/Dockerfile

```diff
@@ -1,5 +1,14 @@
 FROM mhart/alpine-node:16 AS builder
 
+ARG PROXY
+
+# Set up proxy if necessary because yarn does not recognize the corresponding env variables
+RUN if [[ -z $PROXY ]]; then echo "Skip setting up proxy" ; else ( \
```

Comment:
you might want to check if yarn looks for lower or uppercase `HTTP_PROXY`. For some reason these both exist and arent universal

Reply:
I just noticed, that using `docker build .` doesn't have problems with the proxy, but using `docker compose up --build` does 🤔

Reply:
Habs wieder raus genommen, es geht irgendwie auch ohne wenn man `HTTP_PROXY` etc. als build arg von außen definiert